lib/buffer.js

```diff
@@ -335,6 +336,14 @@ Buffer.byteLength = byteLength;
 
 // toString(encoding, start=0, end=buffer.length)
 Buffer.prototype.toString = function(encoding, start, end) {
+  // .toString() can be called automatically on non-Buffers.
```

Comment:
`+Buffer.prototype`, it isn't "supported" but it was decided that it shouldn't kill the process. I can add that to the comment, if you'd like.